app/jobs/fetch_hearing_locations_for_veterans_job.rb

```diff
@@ -69,7 +72,8 @@ def perform
 
           break
         rescue StandardError => error
-          capture_exception(error: error, extra: { appeal_external_id: appeal.external_id })
+          actionable = NONACTIONABLE_ERRORS.include?(error.class)
```

Comment:
should this be the following to the determine if it's actionable?
```suggestion
          actionable = !NONACTIONABLE_ERRORS.include?(error.class)
```

Reply:
Good catch!